homeassistant/components/smappee_official/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "config": {
+        "step": {
+            "pick_implementation": {
+                "title": "Pick Authentication Method"
+            }
+        },
+        "abort": {
+            "already_setup": "Smappee account already configured.",
```

Comment:
The key should be `"already_configured"`. Please use as many common strings as possible. Here are the available common strings:
https://github.com/home-assistant/core/blob/a8e9ccbf1a88d1a9e1c04da36a16910940a9fe19/homeassistant/strings.json#L20-L56

Here's an example:
https://github.com/home-assistant/core/blob/a8e9ccbf1a88d1a9e1c04da36a16910940a9fe19/homeassistant/components/adguard/strings.json#L8